### PR TITLE
document kops deployment

### PR DESCRIPTION
**Pull request by szuecs**
added docs for kops with examples and renew skipper and ingress controller deployments

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: deploy/kops.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+        aws acm describe-certificate --certificate-arn arn:aws:acm:<snip> | jq '.["Certificate"]["Status"]'
+
+If this is no "ISSUED", your certificate is not valid and you have to fix it.
+To resend the CSR validation e-mail, you can use
+
+        aws acm resend-validation-email
+
+
+### Install components kube-aws-ingress-controller and skipper
```
Comment (mikkeloscar) [inline deploy/kops.md]: **kube-ingress-aws-controller**

**Comment by szuecs** [comment]
:+1:

File: deploy/kops.md (hunk 1 above)
Comment (szuecs) [reply deploy/kops.md]: fixed

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**